### PR TITLE
nri-kubernetes: improve helm installation guide and remove custom manifests

### PR DESCRIPTION
**Pull request by roobre**
### Give us some context

This PR adds minor improvements to the Helm installation guide for `nri-kubernetes`, and more notably removes mentions to the so-called "custom manifests" method.

These manifests have been unmaintained for a long period of time, and we do not support their use. The Guided Installer can provide plain manifests, for which support _is_ offered.

### Are you making a change to site code?

Nope

### CODE REVIEW

**Comment by roobre** [comment]
Paging @newrelic/coreint team for preliminary review :)

**Review by gsanchezgavier — COMMENTED** [review]
lgtm thanks!

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 1]
```diff
@@ -34,133 +34,96 @@ kubectl config current-context
...
-    your will.
-
-    ### Installing and configuring `nri-bundle` with Helm
-
-    1. Set the cluster where you want to install the agent:
-
-    ```
-    kubectl config set-cluster <var>DESIRED_CLUSTER</var>
-    ```
+- [`newrelic-infrastructure`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure): Contains the main Kubernetes integration and the infrastructure agent. This is the core component for the New Relic Kubernetes experience.
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:40]: should we mention that is responsible for the cluster explorer?

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 2]
```diff
@@ -34,133 +34,96 @@ kubectl config current-context
...
+- [`nri-prometheus`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-prometheus): New Relic's Prometheus OpenMetrics Integration, allows to scrape Prometheus endpoints and [report metrics to New Relic](https://docs.newrelic.com/docs/integrations/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic#OpenMetrics).
+- [`nri-statsd`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-statsd): New Relic StatsD
+integration.
 
-    To see the available clusters, run `kubectl config get-clusters`.
+Additionally, New Relic provides [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle), a chart which pulls a selectable set of the charts mentioned above. `nri-bundle` can also install Kube State Metrics and Pixie for you if needed.
 
-    2. Add the New Relic Helm charts repo:
+We recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart for Kubernetes deployments, as it provides the easiest way to select, install, and upgrade individual components at your will.
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:50]: I would mention that this is the chart used in the installer for both helm and to generate the manifests

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 3]
```diff
@@ -34,133 +34,96 @@ kubectl config current-context
...
-    --set webhook.enabled=true \
-    --set ksm.enabled=<var>false</var> \
-    --set kubeEvents.enabled=true \
-    --set logging.enabled=true
-    ```
+And switch to the desired context using:
 
-    5. Wait a few seconds, then check that the `DaemonSet` and pods have been created:
+```
+kc config use-context <var>CONTEXT_NAME</var>
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:65]: this is the only reference to kc

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 4]
```diff
@@ -34,133 +34,96 @@ kubectl config current-context
...
+```
+helm repo add newrelic https://helm-charts.newrelic.com
+```
 
-    ### Configuring `nri-bundle` with Helm
+3. Make sure everything is configured properly in the chart by running the following command. Notice that we are specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
 
-    #### Enabling and disabling subcharts
+```
+helm upgrade --install <mark>newrelic</mark> newrelic/nri-bundle \
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:77]: I am not sure it is clear that the wizard will populate the command automatically

File: src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 5]
```diff
@@ -135,448 +130,24 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/k8s-
...
-  </Collapser>
-
-  <Collapser
-    id="non-default-namespace"
-    title="Non-default namespace deployments: Edit config file"
-  >
-    If you want to deploy in a different namespace from `default`, change all values of `namespace` in the manifest.
-  </Collapser>
-
-  <Collapser
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:471]: all the following config options are already described somewhere else?

File: src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 6]
```diff
@@ -135,448 +130,24 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/k8s-
...
-  </Collapser>
-</CollapserGroup>
-
-Here are some additional configurations to consider:
-
-* [Do more configuration for control plane monitoring](/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring)
-* [Link New Relic APM to the Kubernetes integration](/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes)
-* [Monitor services that run on Kubernetes](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#monitor-services)
-
-## Configure the infrastructure agent [#configure-infrastructure-agent]
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:527]: also this is not needed anymore?

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 1 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:40]: Done!

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 3 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:65]: Good catch!

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 4 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:77]: My idea was for this page to be a guide for the "manual installation" using helm. The automated installer should be self-documented.

File: src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx (hunk 5 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:471]: IIRC they should be exposed in the chart `values.yml`

File: src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx (hunk 6 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:527]: I'm not aware of any use case where Kubernetes users wish to modify the agent settings. Can you think about one? We could make a section for "advanced settings" if needed.

Comment (paologallinaharbur) [reply src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:527]: for example for the proxy https://github.com/newrelic/helm-charts/pull/448/files

I guess for fedramp endpoint, however it should be documented as well in the values.yaml

Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:527]: My idea was to duplicate as few info as possible so we don't need to maintain it in two places. We can add it to the readme though!

**Review by paologallinaharbur — APPROVED** [review]
lgtm

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 7]
```diff
@@ -9,7 +9,7 @@ metaDescription: How to install the K8 integration using Helm. Alternative metho
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:9]: the meta description (few lines above) is `How to install the K8 integration using Helm. Alternative method to New Relic's default install for infrastructure monitoring.` However, that is the default now

Comment (paologallinaharbur) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:9]: The page in the dropdown is marked as an alternative
Screen Shot 2021-08-25 at 4 06 54 PM

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 8]
```diff
@@ -34,141 +34,108 @@ kubectl config current-context
...
+kubectl -n <mark>newrelic</mark> get daemonsets,pods
+```
 
-    Please refer to [each of the chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/) and their
-    `values.yml` to see all available configuration options and their defaults.
+6. Make sure you see a `DaemonSet`, and one pod per node for `newrelic-infrastructure`, as well as one `Deployment`
+and one pod for any other component that you have enabled.
 
-  </Collapser>
+### Helm 2 using nri-bundle (recommended)
```
Comment (paologallinaharbur) [inline src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:128]: I would mark it as deprecated and not as recommended

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 7 above)
Comment (paologallinaharbur) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:9]: or is it the alternative with respect to going through the wizard?

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 8 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:128]: Woops, this was certainly a copy-paste artifact. Let me fix that.

File: src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 7 above)
Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:9]: I think "Manual" would be the best choice here, let me change that.

**Comment by roobre** [comment]
Team review done! I think this is now on your capable hands @zuluecho9 :)

**Comment by zuluecho9** [comment]
> Team review done! I think this is now on your capable hands @zuluecho9 :)

I didn't see this @roobre, sorry. Will look soon.

**Review by bradleycamacho — APPROVED** [review]
This all looks great to me! We will file a jira to take a deeper look in a future sprint, but I will go ahead and merge for now. Thanks @roobre !

**Pull request merged**